app/models/poi.rb

```diff
@@ -2,7 +2,8 @@ class Poi < ApplicationRecord
   has_many :trip_pois
   has_many :trips, through: :trip_pois
 
-  validates_presence_of :name,
+  validates_presence_of :name, :population, :state,
```

Comment:
I set up tests for these validations, do you want to nix them all or test all attributes consistently?

Reply:
I think we leave them in.

At they very least they have been helpful for ensuring seeds are in the correct format